qute.ls/com.redhat.qute.ls/src/test/java/com/redhat/qute/parser/template/ASTVisitorTest.java

```diff
@@ -64,4 +65,36 @@ public void visitFromLet() {
 
 	}
 
+	@Test
+	public void visitFromInput() {
+		String content = "{#form uri:Login.manualLogin() id=\"login\"}\r\n" + //
+				"{input name=\"userName\" /}\r\n" + //

```

Comment:
`{input name=\"userName\" /}` is not a valid HTML input, you should not take care of this syntax, the right (html) syntax is:

`<input name=\"userName\" >`